js/app.js

```diff
@@ -0,0 +1,79 @@
+// Section 1
+
+// 1. Concatenation is combining strings. Interpolation is using a variable to represent a value inside a string.
+	// concatenation: console.log("This " + "is " + "a " + "new " + "string.");
+	// interpolation: 
+		// let word = "new";
+		// console.log("This " + "is " + "a " + word + " " + "string.");
+
+// 2. D.R.Y. stands for Don't Repat Yourself. Repeated code is inefficient, so we use loops to avoid repetitive code. 
+
+
+
+// Section 2
+
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+// console.log(a <= b);
+// console.log(c >= d);
+// console.log('Name' === 'Name');
+// console.log(a + b === c);
+// console.log(a * a === d);
+// console.log(e === 'Kevin');
+// console.log(48 == '48');
+
+
+
+// Section 3
+
+// Infinite loop 1: This is not an infinite loop because it doesn't specify what has to be true. 
+
+// Infinite loop II: This is an infinite loop because you can't change the value of a constant, so runProgram = false will give an error and the loop will repeat infinitely because runProgram = true. 
+
+// Reading Code: 
+
+// I think that the output will concatenate "A" onto the string 20 times so the final string will read "AAAAAAAAAAAAAAAAAAA". 
+
+let letters = "A";
+let i = 0;
+while (i<20) {
+	letters += "A";
+	i++;
+}
+
+console.log(letters);
+
+// I ran the code and it gave me 21 As. This is different than what I expected because I forgot that the loop increments after the A has been concatenated. 
+
+
+// Section 4
+
+// For loops
+// The key differences between a for loop and a while loop are that the while loop requires just one condition to be true and can repeat an unknown number of times, whereas a for loop allows you to repeat something for a known number of times and specify the parameters you want to use. 
```

Comment:
good.  a while loop can have a complex control condition though -- based on several AND/OR statements.